### PR TITLE
feat(DENG-997): apple_ads spend per activation

### PR DESCRIPTION
**Pull request by kik-kik**
# feat(DENG-997): apple_ads spend per activation

added a view ad_spend_per_activation view which contains stats about ad campaign performance

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.ad_activation_performance`
+AS
+WITH client_activation_per_campaign AS (
+  SELECT
+    DATE(submission_timestamp) AS `date`,
```
Comment (fbertsch) [inline sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql:6]: Is this right? We have a `submission_timestamp` in `firefox_ios_clients`? What does it represent?

Comment (fbertsch) [inline sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql:6]: Also - there's a 7 day lag from install to when clients activate. We want to associate a use with their install date, not their activation date

File: sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    CAST(REGEXP_EXTRACT(adjust_campaign, r"\s\(([0-9]+)\)") AS INTEGER) AS campaign_id,
+    COUNTIF(is_activated) AS clients_activated,
+    COUNTIF(NOT is_activated) AS clients_not_activated,
```
Comment (fbertsch) [inline sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql:9]: replace this with `new_profiles`

File: sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  client_activation_per_campaign.clients_activated,
+  client_activation_per_campaign.new_profiles,
+  SAFE_DIVIDE(campaign_spend, clients_activated) AS campaign_spend_per_activation,
+FROM
+  ad_stats
+INNER JOIN
+  client_activation_per_campaign
+  ON ad_stats.date_day = client_activation_per_campaign.first_seen_date AND ad_stats.campaign_id = client_activation_per_campaign.campaign_id
+ORDER BY
+  date_day DESC
```
Comment (fbertsch) [inline sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql:55]: nit: ORDER BY seems unnecessary for a view

**Review by fbertsch — APPROVED** [review]
r+wc

File: sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql (hunk 3 above)
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql]: removed, thank you for the review.

**Pull request merged**